modules/couchbase/src/test/java/org/testcontainers/couchbase/Couchbase4_6Test.java

```diff
@@ -0,0 +1,70 @@
+package org.testcontainers.couchbase;
+
+import com.couchbase.client.java.Bucket;
+import com.couchbase.client.java.CouchbaseCluster;
+import com.couchbase.client.java.bucket.BucketType;
+import com.couchbase.client.java.cluster.DefaultBucketSettings;
+import com.couchbase.client.java.document.RawJsonDocument;
+import com.couchbase.client.java.query.N1qlParams;
+import com.couchbase.client.java.query.N1qlQuery;
+import com.couchbase.client.java.query.consistency.ScanConsistency;
+import lombok.Getter;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class Couchbase4_6Test {
```

Comment:
we should probably use Parameterized runner and list all tested versions in one place

Reply:
Or at least create an abstract class with the logic, seems to be repeated